cloudinit/config/cc_vyos_install.py

```diff
@@ -0,0 +1,412 @@
+# VyOS unattended installation module
+# Copyright (C) 2023 VyOS Inc.
+
+# This program is free software; you can redistribute it and/or
+# modify it under the terms of the GNU General Public License
+# as published by the Free Software Foundation; either version 2
+# of the License, or (at your option) any later version.
+
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU General Public License for more details.
+
+# You should have received a copy of the GNU General Public License
+# along with this program; if not, write to the Free Software
+# Foundation, Inc., 51 Franklin Street, Fifth Floor, Boston,
+# MA 02110-1301, USA.
+"""VyOS Installation: Install VyOS unattendedly"""
```

Comment:
I don't think "unattendedly" is a valid adjective. ;)
Should be like "without human intervention" or "in an unattended fashion".